### DISCUSSION TITLE
Events created from manual database insert not showing

### DISCUSSION THREAD

**Discussion by im-perativa** [pretix/pretix#3164] [Q&A]
Hi, I know this is perhaps not the correct way to do it but I want to create hundreds of events quickly. 
I created a sample event from the web UI and looking at the database records created, I copy those records to create new event but I cannot see the new event on the listing page **until I go back to the admin web UI**, open the new event general setting, and click Save. I did not modify any values (based on the record of old event I copy in database) shown there before clicking Save button, 

I manually copy all records associated for an event from these tables:
1. `pretixbase_event`
2. `pretixbase_itemcategory`
3. `pretixbase_item`
4. `pretixbase_event_settingstore`
5. `pretixbase_quota`
6. `pretixbase_quota_items`
7. pretixbase `logentry`

Is there something need to be done after I copy those records? Thank you

**Comment by Akhun-Delar** [comment]
Use the API: https://docs.pretix.eu/en/latest/api/resources/events.html
